test/tool/net/uuidv7_test.lua

```diff
@@ -0,0 +1,25 @@
+-- Copyright 2024 Justine Alexandra Roberts Tunney
```

Comment:
Assuming you want to get your name out there, you are encouraged to own the copyright on your test code, so long as you release it under the same ISC license (see CONTRIBUTING.md). But it's optional. You're perfectly welcome to put my name here too, since I approve of the work you're doing. Thank you for it!

Reply:
That's ok, I copied the uuidv4 one verbatim. I don't need any recognition for that 😆